components/request-card/script.js

```
@@ -0,0 +1,314 @@
+  });
+
+  let oldEndsOnValue = data.oldEndsOn;
+  let newEndsOnValue = data.newEndsOn;
+
+  if (!isExtensionRequest && data.type === 'OOO') {
+    oldEndsOnValue = data.from;
+    newEndsOnValue = data.until;
+  }
+
+  const requestNumber = data.requestNumber || 1;
+  let assigneeNameElement;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'assigneeNameElement' is only assigned here.


(lint/style/useConst)